keel-sql/src/main/resources/db/changelog/20190320-create-lock-table.yml

```diff
@@ -4,7 +4,7 @@ databaseChangeLog:
       author: fletch
       changes:
         - createTable:
-            tableName: lock
+            tableName: cluster_lock
```

Comment:
Testcontainers found another bug: `lock` is a reserved word in MySQL but H2 allowed it.